src/app/components/modal/index.html

```diff
@@ -126,7 +126,7 @@
     <sky-demo-page-property
       propertyName="closed"
     >
-    An event that the modal instance emits when it closes. It emits a <stache-code>SkyCloseModalArgs</stache-code> object with a <stache-code>data</stache-code> property that includes data passed from users on close or save and a <stache-code>reason</stache-code> property that indicates whether the modal was saved or closed without saving.
+    An event that the modal instance emits when it closes. It emits a <stache-code>SkyCloseModalArgs</stache-code> object with a <stache-code>data</stache-code> property that includes data passed from users on close or save and a <stache-code>reason</stache-code> property that indicates whether the modal was saved or closed without saving. The <stache-code>reason</stache-code> property accepts string values such as <stache-code>cancel</stache-code>, <stache-code>close</stache-code>, and <stache-code>save</stache-code>.
```

Comment:
I wonder if would could emphasize it's a string and the examples you listed are examples.  Something like
```
The "reason" property accepts any string value.  Common examples are "cancel," "close," and "save."
```